src/register/components/notification-banner/notification-banner.component.html

```diff
@@ -1,8 +1,6 @@
-<main id="content" role="main">
-  <div aria-labelledby="error-summary-title" class="govuk-error-summary">
-    <h2 class="govuk-error-summary__title">There is a problem</h2>
-    <div class="govuk-list govuk-error-summary__list">
-    <a href="/register-org/register/check#h1"> <ng-content></ng-content></a>
-    </div>
+<div aria-labelledby="error-summary-title" class="govuk-error-summary">
+  <h2 class="govuk-error-summary__title">There is a problem</h2>
```

Comment:
I think this needs to have `id="error-summary-title"` or the `aria-labelledby` property on the surrounding `<div>` will have an accessibility issue.

Reply:
id="error-summary-title" has been added